xcode/Subconscious/Shared/Components/Detail/MemoEditorDetail.swift

```diff
@@ -11,38 +11,36 @@ import ObservableStore
 import Combine
 
 //  MARK: View
-struct DetailView: View {
+struct MemoEditorDetailView: View {
```

Comment:
Name changed to be more descriptive, and to allow multiple types (editor, viewer).